resources/locale/en.yml

```diff
@@ -12,16 +12,30 @@ askvortsov-categories:
       parent_remove_stats: Hide stats for top-level tags?
       parent_remove_last_discussion: Hide most recent discussions for top-level tags?
       small_forum_optimized: Optimize for small forums?
+      enable_primary_tag_color: Enable Primary Tag Color?
```

Comment:
Why don't we always want to enable tag colors?

Reply:
I've shown examples of the theme, which can give admin the choice of having it all minimalistic with the dark and light support.